_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,197 @@
+name: 2020-12-16-newsletter
+slug: 2020-12-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed improvements to static LN backups:** anyone receiving money
+  in a payment channel, including those currently used by LN, needs to
+  keep track of the channel's latest state---all the details that make
+  it possible to close the channel and receive the correct share of the
+  channel's funds onchain.  Unfortunately, computers have a bad habit of
+  losing data and regular periodic backups don't help much when a
+  channel's state could've changed just milliseconds before a disk drive
+  fails.
+
+    LN has always provided some robustness against this type of problem:
+    if your node is offline, your channel counterparty will eventually
+    close the channel so that they can start spending their funds again.
+    This will distribute your funds to the onchain part of your LN
+    wallet, which you hopefully backed up using a normal [BIP32][] seed.
+    This should be reasonably safe: LN's regular penalty mechanism
+    encourages your counterparty to close the channel in it's latest
+    state---if they use an old state, they could lose all their money
+    from the channel.
+
+    The downside of the above approach is that you have to wait for your
+    counterparty to decide you're not coming back.  This wait can be
+    eliminated if you backup some static information about your channel
+    (e.g. the ID of your peer), reconnect to the peer after you lose
+    data, and request the peer immediately close the channel.  This does
+    seem to signal that you've lost data and so your peer can close the
+    channel in an old state, but that isn't guaranteed that will
+    work---you might still have old states that will allow you to
+    penalize them.
+
+    This week, Lloyd Fournier started two threads on the Lightning-Dev
+    mailing list about possible improvements to the above mechanisms:
+
+    - **Fast recovery without backups:** the static per-channel backups
+      that allow fast recovery of funds require you create a new backup
+      each time you open a new channel.  If you fail to make a backup,
+      your only option is to wait until your channel counterparty
+      decides to close the channel on their own.  Fournier instead
+      [proposed][really static backups] a deterministic key derivation method that would allow a
+      node to search through the list of public LN nodes, combine
+      information about its private keys derived from its HD wallet with
+      information about each node's main public key, and determine
+      whether or not it had a channel with that node.  This would only
+      work for channels you open with public nodes, but it's expected
+      that's what most regular people will be using anyway.
+
+    - **Covert state acquisition:** as mentioned, asking your
```

Comment:
I think the following sounds better: 
**Covert state acquisition:**  as mentioned; asking your counterparty to close the channel is evidence that you've lost state and may be vulnerable to theft, but it's not proof. In the event your counterparty attempts to steal from you, they put their own funds at risk. Of course, it'd be better to avoid leaking evidence in the first place. ...